api/Clipboard.json

```diff
@@ -44,53 +44,33 @@
             "web-features:async-clipboard"
           ],
           "support": {
-            "chrome": [
-              {
-                "version_added": "86",
-                "partial_implementation": true,
-                "notes": "From version 86, the <code>text/html</code> MIME type is supported."
-              },
-              {
-                "version_added": "76",
-                "partial_implementation": true,
-                "notes": "From version 76, the <code>image/png</code> MIME type is supported."
-              },
-              {
-                "version_added": "66",
-                "partial_implementation": true,
-                "notes": "Images are not supported."
-              }
-            ],
-            "chrome_android": [
-              {
-                "version_added": "86",
-                "partial_implementation": true,
-                "notes": "From version 86, the <code>text/html</code> MIME type is supported."
-              },
-              {
-                "version_added": "84",
-                "partial_implementation": true,
-                "notes": "From version 84, the <code>image/png</code> MIME type is supported."
-              },
-              {
-                "version_added": "66",
-                "partial_implementation": true,
-                "notes": "Images are not supported."
-              }
-            ],
+            "chrome": {
+              "version_added": "66",
+              "partial_implementation": true,
+              "notes": [
+                "From version 86, the <code>text/html</code> MIME type is supported.",
+                "From version 85, Requires Permission-Policy <code>clipboard-read</code> permission.",
+                "From version 76, the <code>image/png</code> MIME type is supported.",
+                "From version 66, the <code>text/plain</code> MIME type is supported.",
+                "Requires Permission API <code>clipboard-read</code> permission. Does not require transient activation."
+              ]
+            },
```

Comment:
FYI, given that these are all partial implementations, combining the notes like this makes them much easier to use (this is what was done in the other methods). We could consider adding a subfeature for the supported MIME types.

Reply:
I'm a bit unclear why Chrome is marked partial_implementation when other browsers have similar (amounts of) notes and aren't marked partial.

Reply:
Maybe "Does not require transient activation." is the thing that Chrome should have in order to be not-partial?

Reply:
@Elchi3 TBH I did not modify partial and did not really think about it. TLDR; - my take is that probably should make this no longer partial - do you want me to take care of it?

---

I _suspect_ that this was partial originally waiting on `From version 85, Requires Permission-Policy clipboard-read permission.",`.  The spec now no longer requires `clipboard-read` so you could argue it should always have been full implementation. Either way we never remove "full implementation" due to later changes, so it should be full now.

The whole thing is messy because all the other browsers are using transient activation and never intend to implement permissions. The spec has PRs against it to change it this way, but they are unlikely to be accepted without consensus with Chrome. So you might argue that the other browsers are partial implementations against the current spec. I think that just complicates things so we'd best just make them all full.

If you need a more detailed explanation might be best to have a call with me or WillB who have worked through it.

Reply:
> TLDR; - my take is that probably should make this no longer partial - do you want me to take care of it?

yeah, I think we should treat everyone the same at least. Thanks for dealing with the messy compat story here.